recipe/0103-poetry-reading-annotations/annotations.json

```diff
@@ -0,0 +1,48 @@
+{
+   "@context":"http://iiif.io/api/presentation/3/context.json",
+   "id":"{{ id.url }}",
+   "type":"AnnotationPage",
+   "items":[
+      {
+         "id":"{{ id.path }}/canvas/annotation1",
+         "type":"Annotation",
+         "motivation":"commenting",
+         "body":{
+            "type":"TextualBody",
+            "value":"breath",
+            "format":"text/plain"
+         },
+         "target":{
+            "source":"{{ id.path }}/canvas/1",
+            "selector":{
+               "type":"PointSelector",
+               "t":27.660653
+            }
+         }
+      },
+      {
+         "id":"{{ id.path }}/canvas/segment1/annotation/4",
+         "type":"Annotation",
+         "motivation":"commenting",
```

Comment:
Per Rob's comment "Is "her kind" a transcription of the spoken content? If so then it should be motivation supplementing, not commenting. See: https://iiif.io/api/presentation/3.0/#35-values"

Listening to the recording it is a transcription so this should be:
```
"motivation":["supplementing"]
```

Reply:
We'll change this to an array so it matches the IIIF spec rather than the w3c annotation spec.

Commenting is the correct motivation, as below.

Reply:
Sorry I'm wrong here it should be a string...